src/worker/mod.rs

```diff
@@ -246,25 +246,28 @@ impl Worker {
     /// Returns `Ok(None)` in the event of a poll timeout or if the polling loop should otherwise
     /// be restarted
     pub(crate) async fn activity_poll(&self) -> Result<Option<ActivityTask>, PollActivityError> {
```

Comment:
How does this return a Shutdown error now?

Reply:
It's inside activity polling. Will add a test for local-act-only worker being able to shutdown properly as discussed.